github-actions/check-data-assets/README.md

```diff
@@ -1,10 +1,23 @@
-# check-data-assets
+# Gable: Check Data Assets
 
 Checks the data assets specified in the command against their contracts in Gable.
 
 This action is a thin wrapper around the `gable` [Python CLI](https://pypi.org/project/gable/)
 
-```
+## Inputs
+
+| parameter | description | required | default |
+| --- | --- | --- | --- |
+| gable-api-endpoint | Gable API Endpoint in the format https://api.<organization>.gabledata.com | `true` |  |
+| gable-api-key | Gable API Key | `true` |  |
+| gable-version | Gable Version | `false` | latest |
+| allow-gable-pre-release | Whether or not to install pre-release versions of Gable | `false` | false |
+| data-asset-options | Options passed in to the 'gable data-asset check' command  | `true` |  |
+| github-token | Github token used to comment on PR | `false` | ${{ github.token }} |
```

Comment:
is this supposed to show up like this? 
<img width="184" alt="image" src="https://github.com/gabledata/cicd/assets/20962570/40b9671a-2431-4917-85ae-ae4e7f0002c6">

Reply:
Yes I think that was the original author's (@adrianisk's ) intention, to match how it is used ([ex](https://github.com/gabledata/cicd/blob/main/github-actions/check-data-assets/action.yml#L25)).